bin/git-clear

```diff
@@ -1,7 +1,41 @@
 #!/usr/bin/env bash
+PROGNAME="git-clear"
+FORCE=0
 
-echo -n "Sure? - This command may delete files that cannot be recovered, including those in .gitignore [y/N]: "
-read ans
-if [ "$ans" == "y" ] 
-    then git clean -d -f -x && git reset --hard
+function _usage() {
+cat << EOF
+usage: $PROGNAME options
+usage: $PROGNAME -h|help|?
+
+clear git repository
+
+OPTIONS:
+  -f, --force               Force clear without questioning user
+  -h, --help, ?             Show this message
+EOF
+}
+
+# Read arguments
+while [ "$1" != "" ]; do
+    case $1 in
+        -f|--force)
+            FORCE=1
+        ;;
+        -h|--help|?)
+            _usage
+            exit 1
+        ;;
+    esac
```

Comment:
Miss `shift`?

Reply:
Apologies, yep. Added `shift` here in latest commit.